numpy/distutils/mingw32ccompiler.py

```diff
@@ -390,6 +391,13 @@ def _build_import_library_amd64():
                   (out_file))
         return
 
+    # didn't exist in virtualenv, maybe in base distribution?
+    base_file = os.path.join(sys.base_prefix, 'libs', out_name)
+    if os.path.isfile(base_file):
+        log.debug('Skip building import library: "%s" exists' %
+                  (base_file))
```

Comment:
We encourage cleaning up the commit history. PRs are considered private work areas before they are merged.